content/conf.py

```diff
@@ -0,0 +1,106 @@
+github_version = "main"
+conf_py_path = "/content/" # with leading and trailing slash
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    # githubpages just adds a .nojekyll file
+    "sphinx.ext.githubpages",
+    "sphinx_lesson",
+    # remove once sphinx_rtd_theme updated for contrast and accessibility:
+    "sphinx_rtd_theme_ext_color_contrast",
+]
+
+# Settings for myst_nb:
+# https://myst-nb.readthedocs.io/en/latest/use/execute.html#triggering-notebook-execution
+# jupyter_execute_notebooks = "off"
+# jupyter_execute_notebooks = "auto"   # *only* execute if at least one output is missing.
+# jupyter_execute_notebooks = "force"
+jupyter_execute_notebooks = "cache"
+
+# Add any paths that contain templates here, relative to this directory.
+# templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = [
+    "README*",
+    "_build",
+    "Thumbs.db",
+    ".DS_Store",
+    "jupyter_execute",
+    "*venv*",
+]
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "sphinx_rtd_theme"
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['css']
+
+
+# HTML context:
+from os.path import basename, dirname, realpath
+
+html_context = {
+    "display_github": True,
+    "github_user": github_user,
+    # Auto-detect directory name.  This can break, but
+    # useful as a default.
+    "github_repo": github_repo_name or basename(dirname(realpath(__file__))),
+    "github_version": github_version,
+    "conf_py_path": conf_py_path,
+}
+
+# Intersphinx mapping.  For example, with this you can use
+# :py:mod:`multiprocessing` to link straight to the Python docs of that module.
+# List all available references:
+#   python -msphinx.ext.intersphinx https://docs.python.org/3/objects.inv
+# extensions.append('sphinx.ext.intersphinx')
+# intersphinx_mapping = {
+#    #'python': ('https://docs.python.org/3', None),
+#    #'sphinx': ('https://www.sphinx-doc.org/', None),
+#    #'numpy': ('https://numpy.org/doc/stable/', None),
+#    #'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
+#    #'pandas': ('https://pandas.pydata.org/docs/', None),
+#    #'matplotlib': ('https://matplotlib.org/', None),
+#    'seaborn': ('https://seaborn.pydata.org/', None),
+# }
+
+def setup(app):
+    app.add_css_file("style.css")
```

Comment:
I reviewed everything but have some questions about this.  Is this style new for this lesson, or copied from another one?

By now we could add colors to sphinx-lesson directly.  Also, at least to my eye the colors don't look like the best and are somewhat inconsistent (e.g. objectives is orange, questions right above isn't.)  This isn't in sphinx-lesson-template.  I was wondering what's the strategy here?

Still, I would say it is better to merge and proceed and then we can figure out about colors later.  But will leave the merge to you.

Reply:
The colors have been copied from the data visualization lesson. Happy to change this and we can move it more upstream but I just wanted something else than only blue. So I consider this color choice only temporary until this moves more upstream.